### ISSUE TITLE
Implement get products (aliexpress.affiliate.product.query)

### ISSUE THREAD

**Issue by sima96** [sergioteula/python-aliexpress-api#17]
The function get products is not implemented.

The structure of query is writed but in the api.py there isn't the function for use it.

I add in api.py the function get_products that invoke the AliexpressAffiliateProductQueryRequest.

Can i commit it for new version of library?

**Comment by sergioteula** [comment]
Yes sure, you can always pull request and help working on this repo.

**Comment by BackrndSource** [comment]
Done in PR #30
